src/PlaywrightRunner.ts

```diff
@@ -95,15 +96,22 @@ const getDevices = (
   return resultDevices
 }
 
+const getJestTimeout = (configTimeout?: number) => {
+  if (configTimeout) {
+    return configTimeout
+  }
+  return process.env.PWDEBUG ? DEBUG_TIMEOUT : DEFAULT_TEST_PLAYWRIGHT_TIMEOUT
```

Comment:
@mmarkelov I think you should document that you set the default timeout to 15s. I was surprised when running tests after a recent upgrade that it increased from 5 -> 15 s for each jest test. It is worth mentioning in your docs that this is the case.

Also, https://playwright.dev/docs/next/debug/#run-in-debug-mode mentions that it both runs in headful, as well as sets the playwright timeout to 0 (= no timeout). Is this the same you do? I.e. set both Jest's and Playwrights timeout to no timeout? I think it is worth mentioning in the docs what you do when a user sets `PWDEBUG=1`. I.e. do you, in addition to what Playwright does, also change Jest's default?

Reply:
@jhildenbiddle i believe that changes from 5 -> 15 s was made in the first versions of `jest-playwright`, and it was made because some interactions with page can be slow and 5 second was not enough, lots of tests are failing because of it. 
I suppose no timeout for `PWDEBUG` mode is about playwrights timers. And we set timeout to big value, because otherwise the usage of debugging features, like `page.pause()` will relate to jest failure

Reply:
@mmarkelov Did you mean to tag me really? :)

Could you document this? Two things I think of.

1. Document somewhere that you set the default timeout for all tests to 15 sec.
2. Document in the Debug section that when `PWDEBUG` is set, that you following defaults are set: https://playwright.dev/docs/next/debug/#defaults, plus that you set Jest's own timeout to a huge value. Maybe be explicit

I think that would be great :)

Reply:
@thernstig create new issue about doc improvements https://github.com/playwright-community/jest-playwright/issues/602